[CI] Add an option to bring certain tests to the front

jamesr66a opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#24366 [CI] Add an option to bring certain tests to the front**

Differential Revision: [D16815295](https://our.internmc.facebook.com/intern/diff/D16815295)

Commit:
[CI] Run quantization tests first

Commit:
Update on "[CI] Run quantization tests first"


Differential Revision: [D16815295](https://our.internmc.facebook.com/intern/diff/D16815295)

Commit:
Update on "[CI] Run quantization tests first"


Differential Revision: [D16815295](https://our.internmc.facebook.com/intern/diff/D16815295)

ezyang commented:
This doesn't seem sufficient to actually make CI do what you want

Commit:
Update on "[CI] Add an option to bring certain tests to the front"


Differential Revision: [D16815295](https://our.internmc.facebook.com/intern/diff/D16815295)

ezyang commented on test/run_test.py:
```diff
@@ -276,6 +276,16 @@ def parse_args():
         metavar='TESTS',
         help='select the last test to run (excludes following tests)')
+    parser.add_argument(
+        '-b',
```
nit: I probably woudn't give this flag a short form... and if I did it wouldn't be `-b`

ezyang reviewed (approved):
okey dokey

Commit:
Update on "[CI] Add an option to bring certain tests to the front"


Differential Revision: [D16815295](https://our.internmc.facebook.com/intern/diff/D16815295)
